framework/tester.py

```diff
@@ -359,3 +359,17 @@ def wait_all_connections(self, tmt=1):
             if not srv.wait_for_connections(timeout=tmt):
                 return False
         return True
+
+    def start_all_services(self, deproxy: bool = False) -> None:
+        """
+        Start all services.
+
+        Args:
+            deproxy (bool): True - if server is deproxy
+        """
+        self.start_tempesta()
```

Comment:
Is it OK to start Tempesta before servers?

Reply:
I don't know. It works for me. I think this doesn't matter. But I fixed this.